### ISSUE TITLE
Add booleans :(

### ISSUE THREAD

**Issue by VerteXGMD** [megabytesofrem/tsuki#4]
Please add boolean type. Please...

**Comment by megabytesofrem** [comment]
No, strings are better. You can use `str_to_bool` to convert it to a boolean type, but if you want to represent an boolean result from the language you need to use `bool_to_str` /s
